packages/odata/src/interfaces/odataOption.interface.ts

```diff
@@ -55,3 +58,16 @@ export interface OdataOption extends BackendServiceOption {
   /** A callback which will extract and return the dataset from the data queried. Defaults to 'd.results' for v2, 'results' for v3 and 'value' for v4. */
   datasetExtractor?: (response: any) => number;
 }
+
+export interface OdataFilterPredicateArgs {
```

Comment:
should we include any other arguments here? like the gridInstance?

Reply:
the grid instance seems like a good idea, I'm wondering about the `searchValue`, shouldn't this be `searchTerms`? For example a ms-select can have multiple values selected and that's why I've set all filter to use `searchTerms` (plural). But I'm not sure if it was really meant to be typed value in this case here? Another prop might be the `columnFilter` though we can get it from `columnDef.filter`

Reply:
searchTerms is already the parsed result of your regex. that might have implications on how the searchValue gets split. e.g `he*lo` becomes `['he', 'lo']` but it swallows the star that I'd need for my LIKE search as an example if star is the wildcard.

Reply:
ahh ok yeah, if it's the same as the value used by the regex then yeah it's a string, I just wrote it without looking at the code and just making sure it would work with all type of filters :)

Reply:
For my use case I would only need columnDef. I currently also use searchTerms, but I can (better) grab the multi-select values from columnDef.filter.collection.value's.